ios/test_app.rb

```diff
@@ -365,6 +375,14 @@ def use_test_app_internal!(target_platform, options)
           # https://stackoverflow.com/questions/37691049/xcode-compile-flag-to-suppress-nullability-warnings-not-working
           config.build_settings['WARNING_CFLAGS'] ||= ['"-Wno-nullability-completeness"']
         end
+      else
+        # Ensure `ENABLE_TESTING_SEARCH_PATHS` is always set otherwise Xcode may
+        # fail to properly import XCTest
+        unless test_dependencies.assoc(target.name).nil?
+          target.build_configurations.each do |config|
+            config.build_settings['ENABLE_TESTING_SEARCH_PATHS'] = 'YES'
```

Comment:
Maybe it should only set ENABLE_TESTING_SEARCH_PATHS if it wasn't specified at all?  Just in case there's some config that had a good reason for specifying ENABLE_TESTING_SEARCH_PATHS=NO explicitly.